packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -258,7 +258,7 @@ function useBlockEditorSettings( settings, postType, postId ) {
 					}
 				);
 			},
-			__experimentalReusableBlocks: reusableBlocks,
+			__experimentalReusableBlocksSelect,
```

Comment:
I'll make this a private key. This seems something very WP specific right now and seems to require some more thought before exposing an editor setting for user created patterns.